spark-cassandra-connector/src/it/scala/com/datastax/spark/connector/sql/CassandraDataSourceSpec.scala

```diff
@@ -52,14 +52,9 @@ class CassandraDataSourceSpec extends SparkCassandraITFlatSpecBase {
         |CREATE TEMPORARY TABLE $tmpTable
         |USING org.apache.spark.sql.cassandra
         |OPTIONS (
-        | c_table "$table",
+        | table "$table",
         | keyspace "$keyspace",
-        | push_down "$pushDown",
-        | spark_cassandra_input_page_row_size "10",
```

Comment:
We need keep those settings here, change it to spark.cassandra.input.page.row.size to see whether it works or not